wasm.md

```diff
@@ -328,64 +338,64 @@ Note that we do not need to call `#chop` on the results here.
 ```k
     syntax IBinOp ::= "div_u" | "rem_u"
  // -----------------------------------
-    rule <k> ITYPE . div_u I1 I2 => < ITYPE > I1 /Int I2 ... </k> requires I2 =/=Int 0
-    rule <k> ITYPE . div_u I1 I2 => undefined            ... </k> requires I2  ==Int 0
+    rule <k> ITYPE:IValType . div_u I1 I2 => < ITYPE > I1 /Int I2 ... </k> requires I2 =/=Int 0
```

Comment:
I would think the explicit typing is not necessary here, since there is no `div_u` or `div_s` for floats, only `div`, and likewise no `rem_{u|s}`.